src/renderer/components/layout/sidebar.tsx

```diff
@@ -177,6 +178,29 @@ export class Sidebar extends React.Component<Props> {
     });
   }
 
+  renderCluster() {
+    if (!this.clusterEntity) {
+      return null;
+    }
+
+    const { metadata, spec } = this.clusterEntity;
+
+    return (
+      <div className={styles.cluster}>
+        <HotbarIcon
```

Comment:
This component will be refactored in upcoming PRs since it's no `HotbarIcon` anymore but something generic.